src/validate.ts

```diff
@@ -165,32 +223,46 @@ export async function validatePrTitleBump(
       ) {
         throw error;
       }
+      nonCompliantCommitCount += 1;
     }
   }
 
   if (!atLeastOneConventionalCommitFound) {
     core.warning(
       "PR title conforms to Conventional Commits, but none of its commits do."
     );
-    return;
+    return true;
   }
 
   if (highestBump !== prTitle.bump) {
-    const messageList = ` - ${commits.join("\n - ")}`;
+    const commitSubjects = commits.map(m => {
+      return m.split("\n")[0];
+    });
+
+    const nonCompliantCommitsMessage =
+      nonCompliantCommitCount === 0
+        ? ""
+        : "\nPlease note that not all your commits have been found to be compliant; " +
```

Comment:
Can we improve the clarity of this message? (read: Im having difficulties grasping what you intend to say)

Reply:
I'm afraid I'll need your help for that :)

The only thing I can think of to improve this message is to remove the superfluous `this message can be ignored and` part.
So we'd get something like (as an example for a non-matching MINOR bump case):
>Please note that not all your commits have been found to be compliant; if (one of) the non-compliant commit(s) is intended to represent the MINOR bump level, this check will succeed when the message is fixed to be made compliant.

Thoughts?

Reply:
On second thought, the prospect of false positives (and the noise it generates) is worse than just not running the check; I'll remove the check when there is one or more non-compliant commits and replace it with an info-level message.